Sources/VLCSettingsController.m

```diff
@@ -125,12 +138,37 @@ - (void)updateUIAndCoreSpotlightForPasscodeSetting:(BOOL)passcodeOn
 
 #pragma mark - IASKSettings delegate
 
-- (void)settingsViewController:(IASKAppSettingsViewController*)sender buttonTappedForSpecifier:(IASKSpecifier*)specifier
+- (IBAction)pushAboutViewController:(id)sender
 {
     VLCAboutViewController *aboutVC = [[VLCAboutViewController alloc] init];
     [self.navigationController pushViewController:aboutVC animated:YES];
 }
 
+- (UIView *)settingsViewController:(id<IASKViewController>)settingsViewController tableView:(UITableView *)tableView viewForHeaderForSection:(NSInteger)section
+{
+    UIView *header = [[UIView alloc] initWithFrame:CGRectMake(20, 0, tableView.bounds.size.width, 64)];
+    UILabel *headerLabel = [[UILabel alloc] init];
+    
+    headerLabel.font = [UIFont systemFontOfSize:23 weight:UIFontWeightSemibold];
+    headerLabel.text = [settingsViewController.settingsReader titleForSection:section];
+    headerLabel.textColor = PresentationTheme.current.colors.cellTextColor;
+    headerLabel.translatesAutoresizingMaskIntoConstraints = false;
+    [header addSubview:headerLabel];
+    
+    UILayoutGuide *margins = [header layoutMarginsGuide];
+    [[headerLabel.leadingAnchor constraintEqualToAnchor:margins.leadingAnchor constant:8] setActive:true];
```

Comment:
I'm aware that this is a WIP so I'm just going to leave a single comment! 😄 
What would you think about using `[NSLayoutConstraint activateConstraints:@[];`?

Reply:
I didn't know that was a thing! Definitely makes it look cleaner :D

Reply:
If you don't mind, please put more comments on stuff! I'm kinda stuck on most things anyways 🤦‍♂️

Reply:
Yes, and if we trust the [documentation](https://developer.apple.com/documentation/uikit/nslayoutconstraint/1526955-activateconstraints), it's even ~sometimes~ faster! 😄